quora-service/src/main/java/com/upgrad/quora/service/entity/UserEntity.java

```diff
@@ -49,16 +50,17 @@
     @Size(max = 50)
     private String email;
 
+    //@Column(name = "LAST_LOGIN_AT")
+    //private ZonedDateTime lastLoginAt;
+
     @Column(name = "PASSWORD")
     @Size(max = 255)
     @NotNull
-    @ToStringExclude
     private String password;
 
     @Column(name = "SALT")
     @Size(max = 200)
     @NotNull
-    @ToStringExclude
```

Comment:
why we can't use @ToStringExclude ?

Reply:
There were compilation issues. This annotation does not exists.